cmake/googletest.CMakeLists.txt.in

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: BSD-3-Clause "New" or "Revised" License
```

Comment:
This is not a valid license identifier - it should be simply
```suggestion
# SPDX-License-Identifier: BSD-3-Clause
```
Note that the file name in LICENSES has to change as well, obviously.

Reply:
Fixed